pinakes/main/analytics/analytics_collectors.py

```diff
@@ -37,17 +42,16 @@ def config(since, **kwargs):
             "release": platform.release(),
             "type": install_type,
         },
-        # 'install_uuid': settings.INSTALL_UUID,
+        "install_uuid": source.info["install_uuid"],
+        "tower_url_base": source.info["url"],
```

Comment:
@hsong-rh @Ladas This is a good starting point but might not be accessible outside the firewall.